### DISCUSSION TITLE
Isses loading Images after the publish

### DISCUSSION THREAD

**Discussion by Terinax3** [community/community#125355] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am working on a school project and my group encountered an unusual problem after publishing our repository.

We have two images that we need to load on our website, but after publishing, the console reports a 404: failed to load resource error. When I inspect the source code, it appears that our folder containing the images does not exist, despite the folder being present in the GitHub repository.

I noticed that my team mate initialy published from a different branch than the main one. I changed the publishing source to main, but this did not resolve the issue.

As a quick fix, I tried uploading the same images to the JavaScript folder ("js" folder where we store our scripts), but the issue persists—the images simply do not appear.

I haven't encountered this issue before, and I am out of ideas on what could be causing this problem.

I would greatly appreciate anyones insight and help!

**Comment by arashtabaa** [comment, marked as answer]
Step-by-Step Troubleshooting

- Check File Paths in HTML:

`<img src="images/yourimage.png" alt="Description">`
Ensure the path matches the folder structure in your repository.

- Confirm File Existence in Repository:

Navigate to the GitHub repository on the web and manually check if the images are present in the expected directory.

- Check GitHub Pages Settings:

1. Go to your repository on GitHub.
2. Navigate to Settings > Pages.
3. Ensure that the source branch is set to main (or the branch you intend to use).
4. Verify Case Sensitivity:
5. If your image is named MyImage.png, the path should be images/MyImage.png, not images/myimage.png.

- Inspect Network Requests:

1. Open the browser developer tools (usually F12 or right-click > Inspect) and go to the Network tab. Reload your page and look for the 404 errors. The paths listed here can help identify where the issue lies.

- Clearing Browser Cache:

1. On Chrome: Press F12, right-click the refresh button, and select "Empty Cache and Hard Reload".
2. On other browsers, similar options are available in the developer tools.

- Ensure your HTML references look like this:

`<img src="images/image1.png" alt="Image 1">`
`<img src="images/image2.png" alt="Image 2">`
